### PR TITLE
Fix #6 : Creation of PR Template

### LINKED ISSUE #6: Creation of pull request template
Add a pull request template that will help contributors to easily add the description as per our requirements making the entire process seamless.

### PR DESCRIPTION
**Pull request by danishjamal104**
Fixes #6 

Added pull_request_template.md file.
@technojam/app-development team please review this and suggest changes if any.

### CODE REVIEW

**Review by noman2002 — APPROVED** [review]

**Review by AK1406 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Fixes # (issue number)
+
+## Type of change
+Please delete options that are not relevant.
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Checklist:
```
Comment (Sarthak2601) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:13]: A few more checklist points here - 
[ ] - The branch I'm pushing the code from is up-to-date with develop branch. 
[ ] - The title of this Pull Request is in accordance with this format - `Fix #IssueNumber: Small title summarising the change`. 
[ ] - The branch name follows the prescribed format of - `title-of-bug-fixed-or-feature-added`

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] Any dependent changes have been merged and published in downstream modules
+
+## Screenshot
+Add screenshots of UI changes or any new implementations you have made. Incase of no visual change remove this section.
+
+**Mentions**
+- @technojam/app-development
```
Comment (Sarthak2601) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Any particular reason of adding the mentions?

**Review by Sarthak2601 — CHANGES_REQUESTED** [review]
Nice work @danishjamal104. I've suggested a few things below. Thanks!

**Comment by Sarthak2601** [comment]
Also, @danishjamal104 can you please rename this PR to - `Fix #6 : Creation of PR Template`. Thanks!

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 2 above)
Comment (danishjamal104) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Just to ping the team members, nothing specific can be removed. 
What do you suggest?

Comment (danishjamal104) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: I have made the necessary changes just let me know if we should keep Mentions or not. Then I'll commit the new changes.

Comment (Sarthak2601) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: I think we can use the assign and review sections for that. It'll send everyone an email.

**Review by Sarthak2601 — APPROVED** [review]
LGTM. Thanks @danishjamal104 !

**Comment by Sarthak2601** [comment]
Merging this as it has already been approved by everyone.

**Pull request merged**